.formatter.exs

```diff
@@ -1,6 +1,7 @@
+# Used by "mix format"
 [
   inputs: [
-    "{bench,lib,test}/**/*.{ex,exs}",
+    "{bench,config,lib,test}/**/*.{ex,exs}",
```

Comment:
No need to change this file, there is no config directory and I would not expect there to be one in the future.

Reply:
The previous comment still applies, there should be no changes to this file.